pages/linux/brightnessctl.md

```diff
@@ -0,0 +1,24 @@
+# brightnessctl
+
+> Utility for reading and controlling device brightness for GNU/Linux operating systems.
+> More information: <https://github.com/Hummer12007/brightnessctl>.
+
+- List devices with available brightness control:
+
+`brightnessctl --list`
+
+- Get the currently set brightness of the display backlight:
+
+`brightnessctl get`
+
+- Set the brightness of the display backlight to a percenage of the maximum supported value:
+
+`brightnessctl set {{50%}}`
+
+- Increase brightness by certain percent (eg. 10%):
```

Comment:
I think "10%" is clearly shown in the example, no need to mention in description.
```suggestion
- Increase brightness by certain percent:
```